### DISCUSSION TITLE
How to init an Explicit Messaging Connection

### DISCUSSION THREAD

**Discussion by VTA72** [liftoff-sr/CIPster#26] [Q&A]
Hello ! 

I managed to build a class that receive explicite messages initiated by an other device. But I can't find a way to initiate an explicit message connection. How do I declare / reference the remote object ?

Can you give me some hints ?

**Comment by liftoff-sr** [comment]
Do you have access to the EIP specification documents?
Can we move this discussion to the mailing list (on the hope that there may be more participation there)?

**Comment by VTA72** [comment]
Hi Dick !

Thanks for your fast answer, I have access only to the PUB00123R1 found on the ODVA website. I just got the volumes 1 and 2 of CIP.

Let's move the discussion on the mailing list.

Thanks,

With best regards,

Vincent TARDIF
Laboratoire ERTMS France (LEF<https://www.youtube.com/watch?v=ILoUUGF2gxA>)

***@***.***
Découvrez le site Ingénierie du Matériel<https://sncf.sharepoint.com/sites/Sharepoint-MM-MDI/SitePages/Accueil.aspx>

SNCF - DIRECTION DU MATÉRIEL
INGÉNIERIE DU MATÉRIEL - CENTRE d'INGÉNIERIE DU MATERIEL (CIM)
4 Allée des Gémeaux - 72100 LE MANS
MOBILE : +33 (0)6 14 74 36 73
***@***.******@***.***>
Site Internet : https://www.sncf.com/fr/reseau-et-expertises/direction-du-materiel/nos-acteurs

[Rapprochons-nous_RVB]

De : Dick Hollenbeck ***@***.***>
Envoyé : mardi 22 mars 2022 18:59
À : liftoff-sr/CIPster ***@***.***>
Cc : TARDIF Vincent (SNCF VOYAGEURS / DIRECTION DE L'INGENIERIE DU MATERIEL / CIM - LEF) ***@***.***>; Author ***@***.***>
Objet : Re: [liftoff-sr/CIPster] How to init an Explicit Messaging Connection (Discussion #26)


Do you have access to the EIP specification documents?
Can we move this discussion to the mailing list (on the hope that there may be more participation there)?

-
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/liftoff-sr/CIPster/discussions/26*discussioncomment-2416742__;Iw!!Nto2ANp9CeU!Xh4iG7APknEIPTTXFh_7idvN3RJaWi36Ue2_q14RoZ_nWCDDxajXiCCeJAwd39J1Ubvh$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/AYACUXEHLCDJLYACJMAQ7M3VBIC77ANCNFSM5RLHXJFQ__;!!Nto2ANp9CeU!Xh4iG7APknEIPTTXFh_7idvN3RJaWi36Ue2_q14RoZ_nWCDDxajXiCCeJAwd34K--pSI$>.
You are receiving this because you authored the thread.Message ID: ***@***.***>


Interne

-------
Ce message et toutes les pièces jointes sont établis à l'intention exclusive de ses destinataires et sont confidentiels. L'intégrité de ce message n'étant pas assurée sur Internet, la SNCF ne peut être tenue responsable des altérations qui pourraient se produire sur son contenu. Toute publication, utilisation, reproduction, ou diffusion, même partielle, non autorisée préalablement par la SNCF, est strictement interdite. Si vous n'êtes pas le destinataire de ce message, merci d'en avertir immédiatement l'expéditeur et de le détruire.
-------
This message and any attachments are intended solely for the addressees and are confidential. SNCF may not be held responsible for their contents whose accuracy and completeness cannot be guaranteed over the Internet. Unauthorized use, disclosure, distribution, copying, or any part thereof is strictly prohibited. If you are not the intended recipient of this message, please notify the sender immediately and delete it.

**Reply by liftoff-sr** [reply]
Have you joined the mailing list:   cipster@freelists.org ?

This is mentioned in the README file I think.  I was expecting to see you start up a thread there on this topic.
